lib/karma-electron-preprocessor.js

```diff
@@ -19,8 +20,10 @@ function createElectronPreprocessor(karmaBasePath) {
   // Generate our preprocessor function
   function electronPreprocessor(content, file, done) {
     // Render and callback with our content
+    var sourceMaps = convert.fromSource(content);
     var output = template({
-      content: content,
+      content: convert.removeComments(content),
+      sourceMap: sourceMaps !== null ? '\n' + sourceMaps.toComment() : '',
```

Comment:
I'm realizing that we can prob add the source map comment after everything via concatenaiton to `output`. It will prob be cleaner than using the template itself (and necessary if we add double source mapping).

But don't worry, I will write a commit on top of this one to adjust it :+1: